artist-core/src/main/kotlin/com/uber/artist/KotlinArtistCodeGenerator.kt

```diff
@@ -43,11 +44,19 @@ class KotlinArtistCodeGenerator : ArtistCodeGenerator<FileSpec, TypeSpec.Builder
   override val globalTraits: Set<Class<out KotlinTrait>>
     get() = KotlinViewStencilService.newInstance().getGlobalTraits()
 
-  override fun generateFileSpecFor(viewPackageName: String, typeSpecBuilder: TypeSpec.Builder): FileSpec {
+  override fun generateFileSpecFor(viewPackageName: String, typeSpecBuilder: TypeSpec.Builder):
```

Comment:
Add optional empty ArtistRxConfig where we could a list of class
Then loop through the list and add any `addAliasImport`
If we can setup a common config object to pass in these functions this would add help extend the functionality. /

https://sourcegraph.com/github.com/uber/artist/-/blob/artist-traits-rx/src/main/kotlin/com/uber/artist/traits/rx/config/ArtistRxConfig.kt

Reply:
This change looks like a formatting change. If its not uber critical (pun somewhat intended), can we consider this for a future release?